src/game/Pet.h

```diff
@@ -143,8 +143,8 @@ extern const uint32 LevelStartLoyalty[6];
 
 #define ACTIVE_SPELLS_MAX           4
 
-#define PET_FOLLOW_DIST  1.0f
-#define PET_FOLLOW_ANGLE (M_PI_F/2.0f)
+#define PET_FOLLOW_DIST  2.00f
```

Comment:
is that realy an improvement? I dont remeber pet disance problem in current code.